test/CMakeLists.txt

```diff
@@ -159,16 +159,16 @@ if( iodaconv_bufr_ENABLED )
     testinput/bufr_splitting.yaml
     testinput/bufr_filter_split.yaml
     testinput/bufr_adpsfc.yaml
-    testinput/bufr_snow_adpsfc.yaml
-    testinput/bufr_read_2_dim_blocks.yaml
+    testinput/bufr_ncep_snow_adpsfc.yaml
+    testinput/bufr_ncep_read_2_dim_blocks.yaml
```

Comment:
this name change is uneccesary.

Reply:
@emilyhcliu, this branch has conflicts that must be resolved. How to resolve this conflicts? Could you give me a help?

Reply:
> this name change is uneccesary.

Yes, I removed ncep and modified the corresponding CMakeLists.txt. Tested and it works. Now to push to github.